packages/synapse-react-client/src/components/AccessRequirementList/AccessRequirementList.stories.tsx

```diff
@@ -36,31 +31,15 @@ import { getResearchProjectHandlers } from '../../mocks/msw/handlers/researchPro
 const meta: Meta = {
   title: 'Governance/Data Access Request Flow/AccessRequirementList',
   component: AccessRequirementList,
+  parameters: { stack: 'mock' },
   argTypes: {
     isAuthenticated: {
       control: { type: 'boolean' },
-      defaultValue: true,
     },
   },
-  parameters: { stack: 'mock' },
-  decorators: [
-    (Story, args) => (
-      <SynapseContextConsumer>
-        {context => (
-          <SynapseContextProvider
-            synapseContext={{
-              ...context,
-              accessToken: args.isAuthenticated
-                ? context.accessToken ?? 'fake token'
-                : undefined,
-            }}
-          >
-            <Story />
-          </SynapseContextProvider>
-        )}
-      </SynapseContextConsumer>
-    ),
-  ],
+  args: {
+    isAuthenticated: true,
+  },
```

Comment:
I made it much easier to mock the authentication state. In the story, just add `isAuthenticated` as an `arg` and `argType`. The `StorybookComponentWrapper` will take care of the rest.